test/unit/gemm/device/xe_gemm_bf16_bf16_fp32_tensor_op_fp32.cpp

```diff
@@ -42,75 +42,48 @@
 
 #include "gemm_testbed_3x.hpp"
 
-
-TEST(XE_Device_Gemm_bf16t_bf16t_f32t_tensor_op_f32, 256x256x32) {
-  using Config = cutlass::gemm::device::DefaultGemmConfigurationToCutlass3Types<
-    cutlass::arch::OpClassTensorOp, cutlass::arch::IntelPVC,
-    cute::bfloat16_t, cutlass::layout::RowMajor,
-    cute::bfloat16_t, cutlass::layout::RowMajor,
-    float, cutlass::layout::RowMajor,
-    float>;
-
-  using GemmKernel = cutlass::gemm::kernel::GemmUniversal<
+namespace cutlass {
+namespace {
+template <typename LayoutA, typename LayoutB>
+struct XE_Device_Gemm_bf16_bf16_f32_tensor_op_f32 {
+  using Config =
+    cutlass::gemm::device::DefaultGemmConfigurationToCutlass3Types<
+      cutlass::arch::OpClassTensorOp, cutlass::arch::IntelPVC,
+      cute::bfloat16_t, LayoutA,
+      cute::bfloat16_t, LayoutB,
+      float, cutlass::layout::RowMajor,
+      float>;
+
+  using Gemm = cutlass::gemm::device::GemmUniversalAdapter<
+    cutlass::gemm::kernel::GemmUniversal<
       cute::Shape<int,int,int,int>,
-      Config::CollectiveMainloop,
-      Config::CollectiveEpilogue
-  >;
+      typename Config::CollectiveMainloop,
+      typename Config::CollectiveEpilogue>>;
+};
 
-  using Gemm = cutlass::gemm::device::GemmUniversalAdapter<GemmKernel>;
+TEST(XE_Device_Gemm_bf16t_bf16t_f32t_tensor_op_f32, 256x256x32) {
+  using Gemm = XE_Device_Gemm_bf16_bf16_f32_tensor_op_f32<
+    cutlass::layout::RowMajor, cutlass::layout::RowMajor>::Gemm;
```

Comment:
nit: the `cutlass::` isn't needed anymore.

Reply:
thanks. I've removed that.